lib/_sendEvent.ts

```diff
@@ -1,25 +1,9 @@
 import { RequestFnType } from "./utils/request";
-
-export type InsightsEventType = "click" | "conversion" | "view";
-export type InsightsEvent = {
-  eventType: InsightsEventType;
-
-  eventName: string;
-  userToken: string;
-  timestamp?: number;
-  index: string;
-
-  queryID?: string;
-  objectIDs?: string[];
-  positions?: number[];
-
-  filters?: string[];
-};
+import { InsightsEvent } from './types'
 
 export function makeSendEvent(requestFn: RequestFnType) {
   return function sendEvent(
-    eventType: InsightsEventType,
-    eventData?: InsightsEvent
+    ...eventData: InsightsEvent[]
```

Comment:
Considering that you have no special logic around the spread operator and that you pass all the events at once to sendRequest, wouldn't it be better a simpler interface if this accepted an array instead?

Additionally, it might make sense to rename it into `sendEvents`.

This would map closer to what we have in other clients too.

Reply:
You're right. Other public methods like `aa("clickedObjectIDs", { })` needs to keep its signature, so it makes sense to spread there. However, `sendEvent` was never public, I'm already modifying its signature here, so it should be clearer to rename it to `sendEvents` that accepts an array. 👍🏼

Reply:
https://github.com/algolia/search-insights.js/pull/335/commits/673c206505e9284e6df8d10a3baed5e9b14a4174